include/openssl/ssl.h

```diff
@@ -546,8 +546,8 @@ typedef int (*SSL_verify_cb)(int preverify_ok, X509_STORE_CTX *x509_ctx);
 # define SSL_CONF_TYPE_DIR               0x3
 # define SSL_CONF_TYPE_NONE              0x4
 
-/* Length of a TLSv1.3 cookie */
-# define SSL_COOKIE_LENGTH                       255
+/* Maximum length of the application-controlled segment of a a TLSv1.3 cookie */
+# define SSL_COOKIE_LENGTH                       4096
```

Comment:
For some reason I am remembering discussion that changing a defined value in a public header like this would be considered an ABI break; am I misremembering?

Reply:
This is a new define in 1.1.1 so that doesn't apply here.